kt/plugins/godot-intellij-plugin/src/main/kotlin/godot/intellij/plugin/module/GodotModuleBuilder.kt

```diff
@@ -343,6 +343,21 @@ class GodotModuleBuilder : ModuleBuilder(), ModuleBuilderListener {
             outFile.writeText(
                 outFile
                     .readText()
+                    .replace(
+                        "GODOT_MINOR_VERSION",
```

Comment:
I find the name `GODOT_MINOR_VERSION` misleading. I'd prefer just `GODOT_VERSION` or if you want to be expressive `GODOT_VERSION_WITHOUT_PATCH` but that's just a nit.